src/js/oncoprint.js

```diff
@@ -0,0 +1,125 @@
+var Track = require('./track').Track;
+var _ = require('underscore');
+var d3 = require('d3');
+var $ = require('jquery');
+module.exports = {};
+
+// TODO: use self everywhere
+
+var defaultOncoprintConfig = {
+	cell_width: 10,
+	cell_padding: 3,
+	render: 'table',
+};
+
+function Oncoprint(container_selector_string, config) {
+	var self = this;
```

Comment:
Fair enough. There are a few unanswered questions that I have:

1. Why do insert this style in now?
2. I understand that this is possible in the JavaScript programming world, but why introduce it here? What is the utility? What contribution does forcing the user to use `new` (and incurring fatal errors if the user does not) make?